cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 11/7/2023 (PENDING)_
 
+**Features:**
+
+ - Adds Component Testing support for Angular version 17. Addresses [#28153](https://github.com/cypress-io/cypress/issues/28153).
+
```

Comment:
Since this is a feature  we should bump the next version number to 13.5.0, right?

Reply:
Yup